far_mar.rb

```diff
@@ -0,0 +1,3 @@
+module FarMar
+
+end
```

Comment:
Let's require out data classes here so the module knows what classes it contains:

```ruby
module FarMar; end

require './lib/farmar_market.rb'
require './lib/farmar_vendor.rb`
# etc...
```